index.js

```diff
@@ -213,6 +212,8 @@ async function checkHeartbeat() {
 
```

Comment:
Hmmm, what do you think about the WATCHDOG stuff starting on line 152? Should it run if `heartbeatSentAlerts` is `true`? I don't think that it matters, since that WATCHDOG section doesn't actually send any messages to anyone. Which actually makes me doubt the need for me to run that loop over only active locations. I think it would be OK to run over all of them. What do you think? I'm not entirely clear on the reasons for the WATCHDOG state discrepency/reset seeding thing.

Reply:
I intend to scrap the Watchdog altogether in the state machine refactor so my intention was to leave it untouched for now.